### PR TITLE
DOC: Installation instructions via conda

### PR DESCRIPTION
**Pull request by cpelley**
As per title

I have been providing guidance to people installing improver via conda.  Having it defined here makes sense to me so that I can point people to it.

Note the instructions are put together to reflect issues we encountered within actions previously (https://github.com/metoppv/improver/pull/1962).  Mainly, separating the environment containing mamba, with that containing improver.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,34 @@
 [![DOI](https://zenodo.org/badge/85334761.svg)](https://zenodo.org/badge/latestdoi/85334761)
 
 IMPROVER is a library of algorithms for meteorological post-processing and verification.
+
```
Comment (gavinevans) [inline README.md:13]: Perhaps we should also include a Python 3.7 badge above, in addition to the Python 3.6 badge, as the instructions below are for Python 3.7?

**Review by gavinevans — COMMENTED** [review]
Thanks @cpelley 👍 

I've just added a minor comment.

File: README.md (hunk 1 above)
Comment (cpelley) [reply README.md:14]: done 👍

**Comment by cpelley** [comment]
Thanks @gavinevans

**Pull request merged**